source/jormungandr/tests/chaos_disruptions_tests.py

```diff
@@ -484,6 +484,41 @@ def test_disruption(self):
         for disruption in disrupt:
             assert disruption['uri'] != 'test_disruption', 'this disruption must have been deleted'
 
+@dataset([('main_routing_test', ['--BROKER.rt_topics='+chaos_rt_topic, 'spawn_maintenance_worker'])])
+class TestChaosDisruptionsStopPoint(ChaosDisruptionsFixture):
+    """
+    Add disruption on stop point, test if the information is raised in response
+    Then delete disruption, test if there is no more disruptions in response
+    """
+    def test_disruption_on_stop_point(self):
```

Comment:
I think it would be nice to add a test on /journeys for the blocking stop_point (to check that for the moment it's not blocking)